molo/forms/tests/test_views.py

```diff
@@ -775,7 +772,8 @@ def test_article_form_submissions(self):
         url = form.get_full_url()
         data = {
             'article_page': self.article.pk,
-            field.clean_name: 'abc'
+            field.clean_name: 'abc',
+            '': 10
```

Comment:
I see you've added this sort of data to a number of the form submission tests. Why?

Reply:
somewhere it is being added as a required field, the field is blank, I tried looking into it couldn't find where it is coming from. I suspect it is the way we are creating the tests.

Reply:
I _think_ this is what's causing it. This formfield doesn't have a `clean_name` attribute so when it throws the error or accepts the data it probably uses an empty string. If I'm right then setting the `save_article_object` attr on the form should enable you to remove this. 
https://github.com/praekeltfoundation/molo.forms/blob/develop/molo/forms/models.py#L299-L304

Reply:
As discussed on the call, you should probably add the `clean_name` attr to that field when we create it (where I linked above) and then add that to the data instead of these empty fields

Reply:
changed as mentioned